tutorials/crunch-aggregate-sessions.md

```diff
@@ -0,0 +1,296 @@
+---
+layout: page
+title: Aggregating Records with Crunch
+---
+
+## Purpose
+
+The Apache Crunch Java library provides a framework for writing, testing, and running applications in MapReduce pipelines.
+
+This tutorial demonstrates how you use Kite's `CrunchDatasets` helper methods to configure source and target datasets for Crunch operations that aggregate individual events into work sessions.
```

Comment:
As I noted on the other pull request, this should refer back to the section that describes the overall setup for this series of tutorials. I think that context is needed for the reader to know what sessions you are talking about.